python/lsst/pipe/tasks/insertFakes.py

```diff
@@ -486,6 +619,97 @@ def run(self, fakeCat, image, wcs, photoCalib):
 
         return resultStruct
 
+    def _standardizeColumns(self, fakeCat, band):
+        """Use config variables to 'standardize' the expected columns and column
+        names in the input catalog.
+
+        Parameters
+        ----------
+        fakeCat : `pandas.core.frame.DataFrame`
+            The catalog of fake sources to be input
+        band : `str`
+            Label for the current band being processed.
+
+        Returns
+        -------
+        outCat : `pandas.core.frame.DataFrame`
+            The standardized catalog of fake sources
+        """
+        cfg = self.config
+        replace_dict = {}
+
+        # Prefer new config variables over deprecated config variables.
+        # The following are fairly simple to handle as they're just column name
+        # changes.
+        for new_name, depr_name, std_name in [
+            (cfg.ra_col, cfg.raColName, 'ra'),
+            (cfg.dec_col, cfg.decColName, 'dec'),
+            (cfg.bulge_n_col, cfg.nBulge, 'bulge_n'),
+            (cfg.bulge_pa_col, cfg.paBulge, 'bulge_pa'),
+            (cfg.disk_n_col, cfg.nDisk, 'disk_n'),
+            (cfg.disk_pa_col, cfg.paDisk, 'disk_pa'),
+            (cfg.mag_col%band, cfg.magVar%band, 'mag'),
+            (cfg.select_col, cfg.sourceSelectionColName, 'select')
+        ]:
+            # Only standardize "select" column if doSubSelectSources is True
+            if not cfg.doSubSelectSources and std_name == 'select':
+                continue
+            if new_name in fakeCat.columns:
+                replace_dict[new_name] = std_name
+            elif depr_name in fakeCat.columns:
+                replace_dict[depr_name] = std_name
+            else:
+                raise ValueError(f"Could not determine column for {std_name}.")
+        fakeCat = fakeCat.rename(columns=replace_dict, copy=False)
+
+        # Handling the half-light radius and axis-ratio are trickier, since we
+        # moved from expecting (HLR, a, b) to expecting (semimajor, axis_ratio).
+        # Just handle these manually.
+        if (
+            cfg.bulge_semimajor_col in fakeCat.columns
+            and cfg.bulge_axis_ratio_col in fakeCat.columns
+        ):
+            fakeCat = fakeCat.rename(
+                columns={
+                    cfg.bulge_semimajor_col: 'bulge_semimajor',
+                    cfg.bulge_axis_ratio_col: 'bulge_axis_ratio',
+                    cfg.disk_semimajor_col: 'disk_semimajor',
+                    cfg.disk_axis_ratio_col: 'disk_axis_ratio',
+                },
+                copy=False
+            )
```

Comment:
Is there anyway to make this syntax easier to read?

Reply:
Can you clarify?  Which syntax could be improved?

Reply:
I find everything being on different lines very hard to follow. If other people find this spread out format easy to read though then stick with it.

Reply:
I think the indentation style is probably okay.  (FWIW, `black` seems to mostly [agree](https://black.vercel.app/?version=stable&state=_Td6WFoAAATm1rRGAgAhARYAAAB0L-Wj4AHyAMFdAD2IimZxl1N_Wlws4SR3hVF2wOycxJ0Gu6sXMbWebXhHwea0jzbfi2Lnf1krtDagGDPfSJY5xFTLT-ZLMkOGtFwSVShKhu-xhZ_w9UKrYkB9fGu4PFqsx_IbFhPbx7_RoUP5y6DAPWlVJroyRpFtnnLANRdYEKo_C4YYS1wmc1Bf3A6BQGPXNC1QgI2AM4dJhSPJdrM5_FSWZvjUnTPEsYq4NplAzb4k488CsYpU93vhbTKEgOvEnsrUafOLNtbIW5kAAAAAwZM3ITOYJN0AAd0B8wMAAIz1AMaxxGf7AgAAAAAEWVo=).)

But would [more intermediate variables](https://black.vercel.app/?version=stable&state=_Td6WFoAAATm1rRGAgAhARYAAAB0L-Wj4AIUANJdAD2IimZxl1N_Wlb1QRxx4-Na0eP6JSzMZf_ijk0eN6uoEHwHrofLw2sIT4UwmvgqiGKpXFlZUMK1RqJ20LnXcfGmeqYZmRrhENr4gf-qVZgwnnBrEASAHzoqay2JLg3hpEIL8ivvn3rcRE6_CtcS7n7jEoeP7LVjhETdjbHUiNx2ov8FWQiZKR18q9qXs1D2Dad_qHWpFPb7QsQVHqLUjt38pxsJTOkSvr2BK4OefrZSmN_8tJYVgQhNIevSCnaLZwXrsSEJEi8UDKgXMvBYiyMEJAAAAI8Z5DjQxLNYAAHuAZUEAADVPuxfscRn-wIAAAAABFla) help maybe? 

Let me know if you like this or something else better.